packages/core/src/app/common/form/withFormikExtended.test.tsx

```diff
@@ -35,28 +37,32 @@ describe('withFormikExtended', () => {
         mapPropsToValues({ defaultTitle }) {
             return { title: defaultTitle };
         },
-        isInitialValid({ defaultTitle }) {
-            return !!defaultTitle;
-        },
+        enableReinitialize: true,
```

Comment:
Didn't change `enableReinitialize` in production code, only here for testing HOC.